kodilla-rps/src/main/java/com/kodilla/rps/engine/Game.java

```diff
@@ -0,0 +1,18 @@
+package com.kodilla.rps.engine;
+
+import com.kodilla.rps.gameDefinition.GameDefinition;
+import com.kodilla.rps.round.Round;
+
+public class Game {
+
+    private Round round;
```

Comment:
to raczej będzie zmienną lokalną, a nie polem klasy. Ale za to tutaj powinno być `private GameStatistics statistics;`